src/Audis.Primitives/Audis.Primitives.Tests/PrimitivesTests.cs

```diff
@@ -65,23 +64,32 @@ public void TestQuestionIdInvalidFormat(string input, bool throws)
         {
             if (throws)
             {
-                var ex = Assert.Throws<ArgumentException>(() => QuestionId.From(input));
-                Assert.AreEqual($"The QuestionId has an invalid format: \"{input}\", Expected \"<configuration-name>:<lineNumber>\".", ex.Message);
+                var ex = Assert.Throws<ArgumentException>(() => new QuestionId(input));
+                Assert.AreEqual($"The QuestionId has an invalid format: \"{input}\", Expected \"<question-catalog-name>:<lineNumber>\".", ex.Message);
             }
             else
             {
-                var questionId = QuestionId.From(input);
+                var questionId = new QuestionId(input);
                 Assert.AreEqual(input, questionId.Value);
-                Assert.AreEqual(QuestionCatalogName.From(input.Split(':')[0]), questionId.ConfigurationName);
+                Assert.AreEqual(new QuestionCatalogName(input.Split(':')[0]), questionId.QuestionCatalogName);
```

Comment:
Renaming for consistency.

Reply:
Damit ist es jetzt aber wirklich ein breaking change, oder?

Reply:
Ja.

Drei Gründe, die dafür sprechen, das jetzt zu machen:
1. Wir müssen sowieso den Code anpassen aufgrund von `new` vs `.From()`. 
2. Ich glaube wir verwenden das Property sowieso nicht direkt und falls schon, dann nur an ganz wenigen Stellen.
3. Wir releasen jetzt eine Major Version. Da passt das auch vom Semantic Versioning dazu.

Reply:
Alles gute Gründe, aber damit müssen wir alle Verwender updaten, oder?

Reply:
Good Point. Aber ich denke nicht, weil wir den Primitive-Typ trotzdem vom primären Konstruktor über den String-Parameter erzeugen und dann auf die Properties mit `.Split()` aufsplitten. Nur das Property heißt jetzt anders.

Aber stimmt schon, 100% verlassen drauf können wir uns nicht, dass das so ist.